client/app/scripts/liveblog-edit/views/main.html

```diff
@@ -217,7 +217,7 @@ <h3 class="page-nav-title">{{ blog.title }}</h3>
                             </div>
                         </div>
                     </div>
-                    <div class="freetype-selector" ng-click="$event.stopPropagation();" ng-if="selectPostTypeDialog">
+                    <div class="freetype-selector" ng-click="$event.stopPropagation();" ng-if="selectPostTypeDialog" ng-class="{'default': selectedPostType === 'Default'}">
```

Comment:
Is it normal `'default': selectedPostType === 'Default'` is written with a capital D?